draft/draft-ietf-netconf-https-notif.xml

```diff
@@ -473,13 +476,22 @@ INSERT_TEXT_FROM_FILE(../bin/ietf-https-notif-transport@YYYY-MM-DD.yang,69)
           module can be modified by a malicious user to point to an
           invalid URI.</t>
         </list>
+	<list>
+	  <t>The container "receiver-identity" contains nodes like
+	  "cert-maps" that are used by the HTTP server to map to the
+	  HTTPS client's certificate to a 'local' username. A
+	  modification of these nodes will result in the connection
+	  being closed and all notifications in the connection to be
+	  rejected.</t>
```

Comment:
A change will cause an active connection to be closed?  Is this ideal?  I'm not saying it's correct, but many systems apply the new settings to new connections...  

Assuming this is desired behavior, it seems like it should be defined in the body of the draft (or the draft whence it originated), rather than in the Security Considerations section.